app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,49 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" id="mainNav">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+    <a class="navbar-brand js-scroll-trigger" href="/" id="homepage"><img src="/logo_with_name.png" width=auto height="100"></a>
```

Comment:
Added logo name/image instead of text here! It is still a link that goes to the homepage from any page on the app.